CODEOWNERS

```diff
@@ -68,16 +68,16 @@
 /stepactions/fips-operator-check-step-action    @konflux-ci/integration-service-maintainers
 
 # renovate groupName=integration
-/task/coverity-availability-check           @konflux-ci/integration-service-maintainers @kdudka
-/task/coverity-availability-check-oci-ta    @konflux-ci/integration-service-maintainers @kdudka
-/task/sast-coverity-check                   @konflux-ci/integration-service-maintainers @konflux-ci/build-maintainers @kdudka
-/task/sast-coverity-check-oci-ta            @konflux-ci/integration-service-maintainers @konflux-ci/build-maintainers @kdudka
-/task/sast-shell-check                      @konflux-ci/integration-service-maintainers @kdudka
-/task/sast-shell-check-oci-ta               @konflux-ci/integration-service-maintainers @kdudka
-/task/sast-snyk-check                       @konflux-ci/integration-service-maintainers @kdudka
-/task/sast-snyk-check-oci-ta                @konflux-ci/integration-service-maintainers @kdudka
-/task/sast-unicode-check                    @konflux-ci/integration-service-maintainers @kdudka
-/task/sast-unicode-check-oci-ta             @konflux-ci/integration-service-maintainers @kdudka
+/task/coverity-availability-check           @konflux-ci/integration-service-maintainers @kdudka @jeremychoi @sfowl
```

Comment:
I can't invite @jeremychoi @sfowl to https://github.com/orgs/konflux-ci/teams/build-definition-contributors because they are not members of the konflux-ci org yet.

@arewm would you be able to do that?

Reply:
Invites sent.